spec/fabricators/video_fabricator.rb

```diff
@@ -1,4 +1,8 @@
 Fabricator(:video) do
   title { Faker::Lorem.characters(5) }
   description { Faker::Lorem.sentence }
-end
\ No newline at end of file
+end
+
+Fabricator(:video_upload, from: :video) do
+  large_cover { Rack::Test::UploadedFile.new("./spec/support/uploads/monk_large.jpg") }
```

Comment:
Good job on separating it from normal video, but this should work only with Rack test driver. There will be need in the future probably to test it with javascript drivers like Selenium or Poltergeist, or Capybara-webkit. Just keep that in mind.